node/src/node.cpp

```diff
@@ -102,8 +102,9 @@ class control_slot_t:
 
     boost::optional<result_type>
     operator()(const meta_type&, tuple_type&& args, upstream_type&&) {
```

Comment:
BTW, can we make some kind of slot with unwrapped tuple type to deal with Event arguments, not the tuple?